Riot/Modules/Room/RoomViewController.m

```diff
@@ -694,12 +697,14 @@ - (void)viewDidDisappear:(BOOL)animated
 
 - (void)viewWillLayoutSubviews {
     [super viewWillLayoutSubviews];
-    self.shouldScrollToBottomAfterLayout = self.isBubblesTableScrollViewAtTheBottom;
+    self.wasScrollAtBottomBeforeLayout = self.isBubblesTableScrollViewAtTheBottom;
 }
 
 - (void)viewDidLayoutSubviews
 {
     [super viewDidLayoutSubviews];
+    BOOL didViewChangeFrame = !CGRectEqualToRect(lastViewFrame, self.view.frame);
+    lastViewFrame = self.view.frame;
```

Comment:
I do agree that `viewDidLayoutSubviews` can be called for many reasons and very often indeed. Storing the last view frame is fine but I usually store the last bounds of the view as we're interested by the dimension of the view more than its position.